tsconfig.worker.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "target": "es6",
+        "module": "none", // Web workers do not support ES modules

```

Comment:
you can try using `.mts` file extension to tell the tsc compiler that it should use commonjs